src/modules/roc_core/target_musl/roc_core/backtrace.cpp

```diff
@@ -6,15 +6,218 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+#ifndef LIBUNWIND_HH
+#define LIBUNWIND_HH
```

Comment:
Could you please explain why do we need this ifdef?

Reply:
This will prevent any error if libunwind.h included twice somewhere (for example a main file where libunwind.h is included again). Since it doesn't impact the code a lot and libunwind is less likely to be used elsewhere, I will remove it.

Reply:
On my system none of the libunwind headers define `LIBUNWIND_HH`. So even if `libunwind.h` was already included, we will include it here again.

But this should not be a problem because libunwind headers already contain all necessary guards.

The only concern is that we should define `UNW_LOCAL_ONLY` before including `libunwind.h` first time. But since this include is the first in the .cpp file, we can be sure that this is true.